layers/+misc/debug/config.el

```diff
@@ -0,0 +1,8 @@
+(defvar debug-mode 'gdb
+  "Options are `gdb' to run specific debugger.")
+
+(defvar debug-prog nil
+  "Variable that holds the path to the debugger executable.")
+
+(defvar debug-target nil
+  "Executable that should be debugged.")
```

Comment:
You to the point: The idea is to habe it used by other layers in the future. For now I've Python with pdb in my mind. That's why I haven't done them buffer local. But as I'm very new to emacs coding any feedback is welcome.